pom.xml

```diff
@@ -27,7 +27,7 @@
   <properties>
     <revision>1</revision>
     <changelist>999999-SNAPSHOT</changelist>
-    <jenkins.version>2.319.1</jenkins.version>
```

Comment:
TODO: set proper LTS version before merge

Reply:
Just bump to weekly? can always backport important changes?